xlators/protocol/server/src/server-handshake.c

```diff
@@ -122,6 +122,7 @@ do_path_lookup(xlator_t *xl, dict_t *dict, inode_t *parinode, char *basename)
     inode_ref(inode);
 
 out:
+    loc_wipe(&loc);
```

Comment:
`parinode` is assigned to `loc.parent` at line 96 without taking a reference. `loc_wipe()` will release a reference from the `parinode`. Is that ok/expected ?

Reply:
Yes you are correct, we need to take a reference .